src/video/cocoa/cocoa_v.mm

```diff
@@ -125,7 +125,7 @@ - (instancetype)initWithFrame:(NSRect)frameRect andDriver:(VideoDriver_Cocoa *)d
 	this->color_space = nullptr;
 	this->cgcontext   = nullptr;
 
-	this->num_dirty_rects = lengthof(this->dirty_rects);
+	this->dirty_rect = {};
```

Comment:
If this works, maybe do this for the other drivers too, instead of `MemSetT` that is used now?

Reply:
It works :) The rest is bikeshed, it's entirely possible both end up as the same machine code.

Reply:
I like yours more! And for consistency, maybe nice to just make the others look like this too .. just to make my OCD happy :D (I am fine if you don't btw :P)